src/workspaces/NewWorkspaceModal/NewWorkspaceModal.test.ts

```diff
@@ -372,131 +322,9 @@ describe('NewWorkspaceModal', () => {
       }
     );
   });
-  describe('filters billing projects when cloning a workspace ', () => {
```

Comment:
Do we want to maybe replace all this with a test verifying that no azure billing projects show up at all, ever?

Reply:
Added: https://github.com/DataBiosphere/terra-ui/pull/5275/commits/3e3586b7ba7ef55245ac814d6ef35d1906d2e954